lib/datasource/index.js

```diff
@@ -21,6 +22,7 @@ const datasources = {
   github,
   gitlab,
   go,
+  gradle,
```

Comment:
I think this name might be confusing. Can we call it `gradle-wrapper` if that's the only use for it?

Reply:
Soon, gradle will use the "maven" datasource, but that is for maven packages and not for the version of maven itself. We don't really name it like "gradle" = "gradle versions" because in that case "npm" would mean "version of npm", instead of "versions of any npm package". We probably won't have an actual "gradle" datasource but it could be too confusing still to have it named like this. What do you think about renaming gradle -> gradle-version ?